src/main/resources/application.properties

```diff
@@ -5,3 +5,5 @@ spring.datasource.username=
 spring.datasource.password=
 spring.jpa.hibernate.ddl-auto=validate
 spring.jpa.show-sql=true
+jwt.expiration=14400000
+jwt.secret=thisIsVeryImportantAndSecretMessage
```

Comment:
It's better to get sensitive properties from system properties and not pushing it to git.